<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CollectInformation</class>
 <widget class="QDialog" name="CollectInformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CollectInformation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextBrowser" name="NodeInforDisplay"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>节点名称</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>节点前缀</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="NodeName"/>
     </item>
     <item>
      <widget class="QLineEdit" name="NodePrefix"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="AddNodeInfor">
     <property name="text">
      <string>Add node information</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="ButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
